justfile

```diff
@@ -130,30 +130,33 @@ fmt:
     {{ _cargo }} fmt --package {{ _mn_pkg }}
 
 # build a Mnemos binary for the Allwinner D1
-build-d1 board='mq-pro': (_get-cargo-command "objcopy" "cargo-binutils")
+build-d1 board='mq-pro' *CARGO_ARGS='': (_get-cargo-command "objcopy" "cargo-binutils")
```

Comment:
at some point we *may* just want to start turning these just recipes into proper `cargo xtask`s with `mn`, so that we can pass more complex CLI args to them...but i didn't want to do that as part of this change.